### PR TITLE
Fixes #179 - Don't consider HTML formatting in character count limits

### LINKED ISSUE #179: Don't consider HTML formatting in character count limits
If someone includes a link or something like that in the short description (or other fields that have character limits) it's including that as part of the character count and preventing it from saving. It should have a way to consider the max_length without any formatting characters. 

I think we'll have to use a custom max_length validator https://docs.djangoproject.com/en/4.0/ref/validators/

And maybe run this through https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.html.strip_tags.

### PR DESCRIPTION
**Pull request by jonespm**
I changed this class name about 10 times and tested a few different ways. Looks to be the validator pattern for accepting parameters.

### CODE REVIEW

File: backend/canvas_app_explorer/models.py [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
 from db_file_storage.model_utils import delete_file, delete_file_if_needed
 from tinymce.models import HTMLField
 
-# Create your models here.
+# Based on the built in validators 
+# https://github.com/django/django/blob/main/django/core/validators.py
+# Use in your model as validators=[MaxLenIgnoreHTMLValidator(max_length=120)]
+@deconstructible
+class MaxLenIgnoreHTMLValidator:
+    max_length = 120
```
Comment (zqian) [inline backend/canvas_app_explorer/models.py:13]: Works like magic! However, can we consolidate the max_length value definition as a constant? Looks like it is also in the database table definition.

Comment (jonespm) [reply backend/canvas_app_explorer/models.py:13]: Yeah, this is just a default value if one isn't passed in via a parameter. The one used in the model is used. 

I was thinking afterward of just raising an exception if it wasn't set or maybe just extending the [MaxLengthValidator](https://github.com/django/django/blob/49b470b9187b6be60e573fed08c8f4a87f133750/django/core/validators.py#L433) that's there.

File: backend/canvas_app_explorer/models.py [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+from django.utils.html import strip_tags
 from db_file_storage.model_utils import delete_file, delete_file_if_needed
 from tinymce.models import HTMLField
 
-# Create your models here.
+# Based on the built in validators 
+# https://github.com/django/django/blob/main/django/core/validators.py
+# Use in your model as validators=[MaxLenIgnoreHTMLValidator(max_length=120)]
+@deconstructible
+class MaxLenIgnoreHTMLValidator:
```
[the remark this answers is not in the document]
Comment (ssciolla) [reply backend/canvas_app_explorer/models.py:12]: Maybe you could simplify here by extending `BaseValidator` like `MaxLengthValidator` does?
https://github.com/django/django/blob/49b470b9187b6be60e573fed08c8f4a87f133750/django/core/validators.py#L432-L447

**Review by ssciolla — APPROVED** [review]
This seems to work, so I'm approving. I had some refactoring/text change suggestions, do with them as you see fit.

File: backend/canvas_app_explorer/models.py (hunk 1 above)
Comment (jonespm) [reply backend/canvas_app_explorer/models.py]: This is outdated with the current code and no longer used.

**Review by ssciolla — APPROVED** [review]

**Pull request merged**